tests/unit/test_ocsort.py

```
@@ -0,0 +1,181 @@
+    [
+        {"det_thresh": 1.5},
+        {"det_thresh": -0.1},
+        {"iou_threshold": 2.0},
+        {"max_age": -1},
+        {"min_hits": -1},
+        {"delta_t": 0},
+        {"inertia": -0.5},
+    ],
+)
+def test_config_validation_rejects_bad_values(kwargs):
+    with pytest.raises(ValueError):
```

Ruff (0.15.15)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)